custom-nb-image/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN pip install -r requirements.txt
 RUN pip uninstall pickle5 -y
 
 # Install codeflare-sdk and other libraries
```

Comment:
I think the ARG needs to be moved here so that layers are reused when building.

See: https://stackoverflow.com/questions/41593135/does-putting-arg-at-top-of-dockerfile-prevent-layer-re-use